### PR TITLE
ci-operator/config: update config for cri-o release-1.28

### PR DESCRIPTION
**Pull request by sohankunkerkar**

### CODE REVIEW

File: ci-operator/config/cri-o/cri-o/cri-o-cri-o-release-1.28.yaml [hunk 1]
```diff
@@ -0,0 +1,68 @@
+base_images:
+  base-machine:
+    name: centos
+    namespace: openshift
+    tag: stream9
+  rhel-coreos-9-base:
+    name: "4.15"
+    namespace: ocp
+    tag: rhel-coreos-9
```
Comment (haircommander) [inline ci-operator/config/cri-o/cri-o/cri-o-cri-o-release-1.28.yaml:9]: can you also update the [main.yaml(https://github.com/openshift/release/blob/master/ci-operator/config/cri-o/cri-o/cri-o-cri-o-main.yaml) file to match this?

Comment (sohankunkerkar) [reply ci-operator/config/cri-o/cri-o/cri-o-cri-o-release-1.28.yaml:9]: Sure, added

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-setup-periodic

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-setup-periodic

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-setup-periodic

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-setup-periodic

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-setup-periodic

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-setup-periodic

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-setup-fedora-periodic

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-setup-periodic

**Review by haircommander — APPROVED** [review]

**Pull request merged**